<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaunchDialog</class>
 <widget class="QDialog" name="LaunchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>366</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>366</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图与网络优化</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="btnAbout">
     <property name="text">
      <string>关于</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnShortestPath">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>51</height>
        </size>
       </property>
       <property name="text">
        <string>最短路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNSM">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>51</height>
        </size>
       </property>
       <property name="text">
        <string>网络单纯形法</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
